servermaster/server.go

```diff
@@ -611,6 +611,10 @@ func (s *Server) runLeaderService(ctx context.Context) (err error) {
 	defer func() {
 		s.leader.Store(&Member{})
 		s.resign()
+		err := s.jobManager.Close(ctx)
```

Comment:
could you please explain why this leads to a panic ?  In the jobmanager close method ,just closing the message topic

Reply:
If we don't close job manager, the message handler is remained in messageServer, which is a singleton in the server master node.